src/watchdog/mongoose_watchdog.erl

```diff
@@ -0,0 +1,236 @@
+%%%-------------------------------------------------------------------
+%%% @author denys
```

Comment:
I think we should do either of:

1. Not add this kind of comment at all. My preferred option, there's `git log` to check authors, commit dates & stuff, there's a COPYING file in the repo for the license.
1. Actually make it useful and binding wrt copyright: use a `First Last <email@company>` for `@author` (that's what git uses, that's the minimum info required for people outside the team), use the actual company name instead of `<COMPANY>` for `@copyright`, use the current year - 2019, and actually write something in `@doc`.

Reply:
i would rather want to have local readme.md file for this functionality than put it in the @doc section.

Reply:
I guess it wouldn't hurt to leave a one or two sentence long summary in the `@doc` section and point to more details in the README.